packages/react-search-ui/src/containers/ResultsPerPage.js

```diff
@@ -12,7 +13,9 @@ export class ResultsPerPageContainer extends Component {
     // State
     resultsPerPage: PropTypes.number.isRequired,
     // Actions
-    setResultsPerPage: PropTypes.func.isRequired
+    setResultsPerPage: PropTypes.func.isRequired,
+    results: PropTypes.arrayOf(ResultType),
```

Comment:
This is similar to Sorting.js container:
> Sorry if I'm wrong, but why we need to exclude results and searchTerm from the ...rest here?
It looks like for this file we could remove them both from this.props destructuring and from propTypes.
Does it make sense?